packages/core/codeframe/test/fixtures/wordpress-components.js

```diff
@@ -0,0 +1,39 @@
+import _extends from "@babel/runtime/helpers/esm/extends";
+import _objectWithoutProperties from "@babel/runtime/helpers/esm/objectWithoutProperties";
+import { createElement } from "@wordpress/element";
+
+/**
+ * External dependencies
+ */
+import { unstable_CompositeItem as CompositeItem } from 'reakit/Composite';
```

Comment:
Can you remove everything below line 10? Technically this is license infringement, but just with the imports, it should be fine.

Reply:
Just replaced it with dummy code, the bug had nothing to do with the actual code anyway